sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.44.1-preview</Version>
+		<Version>1.45.1-preview</Version>
```

Comment:
lol I either am working on updating 1.44.1 to 1.45. Maybe here should be 1.45.0 rather than 1.45.1?

Reply:
good point

Reply:
And it is still -preview? If it is a revision of preview you may want to consider 1.44.1-preview.1/2/3 etc.